misc/docs/source/tutorial/code_snippets/beachball_plot2.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import io
+import numpy as np
+import matplotlib.pyplot as plt
+from obspy.imaging.beachball import beach
+
+urls = ['https://raw.githubusercontent.com/carltape/mtbeach/refs/heads/master/gmt/dfiles/beachballs_ipts1_iref1_lune_psmeca','https://raw.githubusercontent.com/carltape/mtbeach/refs/heads/master/gmt/dfiles/beachballs_ipts1_iref2_lune_psmeca','https://raw.githubusercontent.com/carltape/mtbeach/refs/heads/master/gmt/dfiles/beachballs_ipts1_iref3_lune_psmeca','https://raw.githubusercontent.com/carltape/mtbeach/refs/heads/master/gmt/dfiles/beachballs_ipts1_iref4_lune_psmeca','https://raw.githubusercontent.com/carltape/mtbeach/refs/heads/master/gmt/dfiles/beachballs_ipts1_iref5_lune_psmeca']
```

Comment:
are these files super heavy ? if not, I think it'd be better to include them, as we can't really rely on their persistence "forever" ?